packages/cms-cli/commands/ls.js

```diff
@@ -0,0 +1,131 @@
+const {
```

Comment:
I wonder if we should call the command `list` and make `ls` an alias sort of like what we have done with `remove` and `rm`.